src/muse/sectors/sector.py

```diff
@@ -138,20 +153,26 @@ def __init__(
 
         :py:mod:`muse.interactions` contains MUSE's base interactions
         """
+        if interactions is None:
+            interactions = interaction_factory()
+        self.interactions = interactions
```

Comment:
Merely stylistic, so up to you, but I tend to use in this cases:
```suggestion
        self.interactions = interactions or interaction_factory()
```
But I'm not entirely sure if this works in older Python versions...

Reply:
I agree this is way better